op-challenger/flags/flags.go

```diff
@@ -0,0 +1,79 @@
+package flags
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+
+	opservice "github.com/ethereum-optimism/optimism/op-service"
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	opmetrics "github.com/ethereum-optimism/optimism/op-service/metrics"
+	oppprof "github.com/ethereum-optimism/optimism/op-service/pprof"
+	oprpc "github.com/ethereum-optimism/optimism/op-service/rpc"
+	txmgr "github.com/ethereum-optimism/optimism/op-service/txmgr"
+)
+
+const envVarPrefix = "OP_CHALLENGER"
+
+var (
+	// Required Flags
+	L1EthRpcFlag = cli.StringFlag{
+		Name:   "l1-eth-rpc",
+		Usage:  "HTTP provider URL for L1.",
+		EnvVar: opservice.PrefixEnvVar(envVarPrefix, "L1_ETH_RPC"),
+	}
+	RollupRpcFlag = cli.StringFlag{
+		Name:   "rollup-rpc",
+		Usage:  "HTTP provider URL for the rollup node.",
+		EnvVar: opservice.PrefixEnvVar(envVarPrefix, "ROLLUP_RPC"),
+	}
+	L2OOAddressFlag = cli.StringFlag{
+		Name:   "l2oo-address",
+		Usage:  "Address of the L2OutputOracle contract.",
+		EnvVar: opservice.PrefixEnvVar(envVarPrefix, "L2OO_ADDRESS"),
+	}
+	DGFAddressFlag = cli.StringFlag{
+		Name:   "dgf-address",
+		Usage:  "Address of the DisputeGameFactory contract.",
+		EnvVar: opservice.PrefixEnvVar(envVarPrefix, "DGF_ADDRESS"),
+	}
+	PrivateKeyFlag = cli.StringFlag{
```

Comment:
The tx manager already has a flag for the private key

Reply:
Ah right! I believe I originally had it here in my isolated repo so i could force it to be `required`, but will remove since we don't want to enforce this requirement.